packages/fluentui/react-northstar/src/themes/teams/components/Input/inputLabelVariables.ts

```diff
@@ -0,0 +1,15 @@
+import { pxToRem } from '../../../../utils';
+
+export interface InputLabelVariables {
+  insideLabelBottom: string;
+  insideLabelPaddingLeft: string;
+  insideLabelActiveFontSize: string;
+  inlineLabelPaddingRight: string;
+}
+
+export const inputLabelVariables = (siteVars): InputLabelVariables => ({
+  insideLabelBottom: pxToRem(-8),
+  insideLabelPaddingLeft: pxToRem(12),
+  insideLabelActiveFontSize: pxToRem(12),
+  inlineLabelPaddingRight: pxToRem(10),
+});
```

Comment:
Should be this defined under `inputVariables` and then a reexport be used there?

Reply:
Dunno, should it?  I created this variable file after being pointed by @mnajdova